### DISCUSSION TITLE
ea和epic

### DISCUSSION THREAD

**Discussion by zxxiao111** [share121/inter-knot#3288] [其他]
大家好。我来求助了。我以前的steam等账号用的都是谷歌邮箱。而现在谷歌又不了了我便想把邮箱变更成QQ邮箱但是ea和epic都要原邮箱验证所以我不知道怎么办前来求助

**Comment by RaphaelHider** [comment]
为什么用不了谷歌呢

**Reply by Ziew-Ryan** [reply]
会不会是因为谷歌之前宣布的新账号安全政策，一年之内都不活跃的账号和邮箱都会被注销

**Reply by iuhjk** [reply]
真的假的，我得上去瞅瞅

**Comment by zxxiao111** [comment]
不是，因为我很久没用了不知道怎么回事。然后再一次偶然的情况下我下载回来了epic然后验证邮箱账号的时候收不到短信，然后我就重新登陆了结果显示地区不支持

**Reply by likuol** [reply]
开个梯子估计就可以了
